cfme/tests/infrastructure/test_host.py

```diff
@@ -541,9 +541,10 @@ def test_add_ipmi_refresh(appliance, setup_provider):
 
 
 @test_requirements.infra_hosts
-@pytest.mark.parametrize("num_hosts", [1, 2, 4])
-@pytest.mark.meta(blockers=[BZ(1634794, forced_streams=["5.10"])], automates=[1634794])
-def test_infrastructure_hosts_crud(appliance, setup_provider_min_hosts, provider, num_hosts):
+@pytest.mark.meta(automates=[1634794])
+@pytest.mark.parametrize("crud_action", ['edit_from_hosts', 'edit_from_details', 'cancel',
+                                         'nav_away_changes', 'nav_away_no_changes', 'remove'])
```

Comment:
In my opinion, and definitely just an opinion so feel free to disagree, the `crud` test should be only testing core functionality, i.e. can you create, update, and delete an object? The parametrization here is only around the `update` part, and therefore I think it should be in a separate test like `test_infrastructure_host_update_scenarios`. 

Then the crud test would only include the basic `update` scenario. What do you think?

Reply:
I disagree. I don't really have knowledge of the history of the usage of CRUD within CFME, but I  am only testing that we edit and delete here like I would expect. there are a few variations to deal with the different places you can edit from. It seemed like a natural place to add these. If this is really something the team discourages, I could make a change, but I'd question why we are getting so particular with the naming of the test. Isn't coverage the end goal? What would splitting the tests up accomplish? I parameterize around edit and delete functions, not just edit.

Reply:
I could split this up, I just want to make sure this is what we want to do, It seems to me I am just making 2 tests out of one where I see much of the time we are trying to combine tests AFAICT.

Reply:
@john-dupuy thanks for voicing the opinion, but I think the parametrization of the CRUD test function is a good way to consolidate the test coverage here, as long as the parametrization is focused on CRUD paths.

For example, I wouldn't want the CRUD parametrization to include the credentials update coverage, or any comparisons.

I think that including edit via details / all selection and delete via the same selection options is appropriate to include in CRUD parametrization, because they're just the different ways that the entity can be updated/deleted from MIQ. To me this equivalent to parametrization of the ViaUI/ViaREST contexts for CRUD.